dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+                    continue;
+                }
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileTitle=\"")
+                    .append(UtilMethods.espaceForVelocity(fileName))
+                    .append("\" )");
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileSize=\"")
+                    .append(UtilMethods.espaceForVelocity(filesize))
+                    .append("\" )");
+                String binaryFileURI = fileName.length() > 0
+                        ? UtilMethods.espaceForVelocity("/contentAsset/raw-data/" + content.getIdentifier() + "/"
+                                + field.variable() + "/" + content.getInode())
+                        : "";
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileURI=\"")
+                    .append(binaryFileURI)
+                    .append("\" )");
+            } else if (field instanceof SelectField) {
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("SelectLabelsValues=\"")
+                    .append(field.values()
+                        .replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\")");
+            } else if (field instanceof RadioField) {
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("RadioLabelsValues=\"")
+                    .append(field.values()
+                        .replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\" )");
+            } else if (field instanceof CheckboxField) {
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("CheckboxLabelsValues=\"")
+                    .append(field.values()
+                        .replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\" )");
+            } else if (field instanceof DateField) {
+                String shortFormat = "";
+                String dbFormat = "";
+                if (contFieldValueObject != null && contFieldValueObject instanceof Date) {
+                    shortFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "MM/dd/yyyy");
+                    dbFormat = UtilMethods.dateToHTMLDate((Date) contFieldValueObject, "yyyy-MM-dd");
+                }
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("=$date.toDate(\"yyyy-MM-dd\", \"")
+                    .append(dbFormat)
+                    .append("\"))");
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("ShortFormat=\"")
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ShortFormat=\"" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)